app/konnect/dev-portal/access-and-approval/add-teams.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Add Developer Teams from Identity Providers
+content_type: how-to
+---
+
+In the {{site.konnect_short_name}} Dev Portal, you can import existing developer teams from a third-party identity provider (IdP) and map their permissions to elements in {{site.konnect_short_name}}. This allows you to import developers automatically instead of manually copying over every single team of developers from your IdP to {{site.konnect_short_name}}.
+
+This guide explains how to map the permissions, including scopes and claims, from your group of developers in your IdP to your organization's team in {{site.konnect_short_name}}. Although this guide uses Okta, Azure Active Directory (AD), and Auth0 as examples, you can use any IdP that conforms to OIDC standards. 
+
+## Prerequisites
+
+* A test developer account in your IdP
+* An IdP configured in Konnect (OIDC or DCR) for dynamic client registration:
```

Comment:
@c3pko You mentioned copying over instructions from the DCR guides, but I decided to add them as a prerequisite with links to those docs just so we don't have to maintain content in multiple docs.

Reply:
I think we should keep this section in step 6 because it's the meat and potatoes of how to get team mapping working. my comment was more about the language being corrected to be in the context of an application in your IdP, not your IdP. e.g. these fields

Provider URL: The value stored in the issuer variable for your IdP.
Client ID: The application ID for your IdP.
Client Secret: The client secret of your IdP.
Scopes: The scopes to be requested from the OpenID Provider.


should all read something along the lines of "from your application in your IdP" not "for your IdP"

Reply:
@c3pko Ah, I think I understand. I revised the wording in step 6, I think as long as that looks good, this should be ready to go!